utility/servicer/module_test.go

```diff
@@ -11,25 +15,53 @@ import (
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/modules"
 	mockModules "github.com/pokt-network/pocket/shared/modules/mocks"
+	typesUtil "github.com/pokt-network/pocket/utility/types"
 )
 
-var testServicer1 = &coreTypes.Actor{
-	ActorType: coreTypes.ActorType_ACTOR_TYPE_SERVICER,
-	Address:   "a3d9ea9d9ad9c58bb96ec41340f83cb2cabb6496",
-	PublicKey: "a6cd0a304c38d76271f74dd3c90325144425d904ef1b9a6fbab9b201d75a998b",
-	Chains:    []string{"0021"},
-}
+const (
```

Comment:
Can you create a ticket to add extensive E2E tests for this. See e2e/tests/root.feature

Reply:
Done. #833